pkg/operator/ceph/file/mds/mds_liveness_probe.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+ROOK_CEPH_MON_HOST="{{ .RookCephMonHost }}"
+ROOK_CEPH_MON_INITIAL_MEMBERS="{{ .RookCephMonInitialMembers }}"
+DAEMON_MDS="{{ .DaemonMds }}"
+FILESYSTEM_NAME="{{ .FilesystemName }}"
+
+outp="$(ceph fs dump --mon-host="$ROOK_CEPH_MON_HOST" --mon-initial-members="$ROOK_CEPH_MON_INITIAL_MEMBERS" --format json 2>&1)"
+rc=$?
+if [ $rc -ne 0 ]; then
+    echo "ceph mds dump check failed with the following output:"
+    echo "$outp" | sed -e 's/^/> /g'
+    echo "we couldn't get the information to reliably tell if the MDS should be restarted or not"
+    return 0
+fi
+
+# get the active and standby mds in the fs map
+standbyMds=$(echo "$outp" | jq ".standbys|map(.name)")
+activeMds=$(ceph fs get "$FILESYSTEM_NAME" --mon-host="$ROOK_CEPH_MON_HOST" --mon-initial-members="$ROOK_CEPH_MON_INITIAL_MEMBERS" --format json 2>&1 | jq ".mdsmap.info|map(.name)")
```

Comment:
What does `ceph fs get` return that `ceph fs dump` doesn't? Can we switch to using only one of these commands to keep things as simple and straightforward as possible?

Reply:
We changed it as jq will need to query a lot, I can revert it back too

Reply:
And I think filesystem is not the unique Key, so we need to query the filesystems array and check it again and again